blackbox/docs/sql/administration/privileges.txt

```diff
@@ -110,6 +110,13 @@ privilege on all the other tables of the ``doc`` schema.
 Behavior of ``GRANT``, ``DENY`` and ``REVOKE``
 ==============================================
 
+.. note::
+
+    Stale permissions might be introduced if ``DDL`` statements were invoked
+    while the Enterprise Edition is temporarily disabled. To allow clients
+    to remove such stale permissions, the ``REVOKE`` statement does not perform
+    any validation checks on the table or schema names.
```

Comment:
I think we should mention the effect of this here. eg: This enables the user to drop the stale privileges even if the table does not exist anymore. 
Or something along those lines. 
Also I don't think we should mention schemas here (as it implies that the other statements, like `GRANT` do some schema validation)